examples/lighting-app/mbed/README.md

```diff
@@ -0,0 +1,225 @@
+        -   [Notes](#notes)
+-   [Device UI](#device_ui)
+ 
+
+<hr> 
+
+
+<a name="overview"></a>
+## Overview
+
+
+The CHIP device that runs the lighting application is controlled by the CHIP
+controller device over WIFI. By default, the CHIP device is disconnected
+, and it should be paired with CHIP controller and get
+configuration from it. Actions required before establishing full
+communication are described below.
+
+<a name="bluetooth-le-advertising"></a>
+### Bluetooth LE advertising
+
+To commission the device onto a CHIP network, the device must be discoverable
+over Bluetooth LE. . The Bluetooth LE advertising starts
+automatically after device boot-up.
+
+<a name="bluetooth-le-rendezvous"></a>
+### Bluetooth LE rendezvous
+
+In CHIP, the commissioning procedure (called rendezvous) is done over Bluetooth
+LE between a CHIP device and the CHIP controller, where the controller has the
+commissioner role.
+
+To start the rendezvous, the controller must get the commissioning information
+from the CHIP device. The data payload is encoded within a QR code, printed to
+the UART console.
+
+<a name="wifi_provisioning"></a>
+#### WiFi provisioning
+
+Last part of the rendezvous procedure, the provisioning operation involves
+sending the  network credentials from the CHIP controller to the CHIP
+device. As a result, device is able to join the  network and communicate
+with other  devices in the network.
+
+
+
+<a name="building_flashing"></a>
+# Building and Flashing
+
+<a name="environment_setup"></a>
+## Development environment setup
+
+Before building the example, check out the CHIP repository and sync submodules
+using the following command:
+
+        $ git submodule update --init
+
+Building the example application requires the use of **ARM Mbed-OS** sources 
+and **arm-none-gnu-eabi** toolchain. The OpenOCD package is used for 
+flashing purpose.
+<br>
+Some additional packages may be needed, depending on selected build target 
+and its requirements.
+
+> **The VSCode devcontainer has these components pre-installed. Using 
+the VSCode devcontainer is the recommended way to interact with ARM Mbed-OS 
+port of the CHIP Project.**    
+>    
+> **Please read this [README.md](../../..//docs/VSCODE_DEVELOPMENT.md) for 
+more information about using VSCode in container.**
+To initialize development environment, download all registered sub-modules 
+and activate development environment:
+```
+$ source ./scripts/bootstrap.sh    
+$ source ./scripts/activate.sh
+```
+If packages are already installed then simply just activate development 
+environment:
+```
+$ source ./scripts/activate.sh
+```
+<a name="building"></a>
+## Building
+
+The Lighting-app application can be built in the same way as any other CHIP 
+example ported to mbed-os platform.
+<br>
+* **by using generic vscode task**:  
+```
+Terminal => Run Task... => Run Mbed application => build => (example name) => (board name) => (build profile)`    
```

Comment:
It's not the terminal it is the Command Palette accessible with `F1`. So maybe mention `Command Palette (F1)` instead of terminal.

Reply:
Done